### PR TITLE
provider/consul: Migrate to using on-host consul agent

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

**Comment by lwander** [comment]
@duftler relates to https://github.com/spinnaker/spinnaker/pull/1009

File: clouddriver-consul/src/main/groovy/com/netflix/spinnaker/clouddriver/consul/config/ConsulConfig.groovy [hunk 1]
```diff
@@ -41,8 +41,12 @@ class ConsulConfig {
       agentPort = 8500 // Default used by consul
     }
 
-    if (!servers) {
-      throw new IllegalArgumentException("Consul servers must be provided.")
+    if (!agentEndpoint) {
+      agentEndpoint = 'localhost'
+    }
+
+    if (agentEndpoint.indexOf(":") == -1) {
```
Comment (duftler) [inline clouddriver-consul/src/main/groovy/com/netflix/spinnaker/clouddriver/consul/config/ConsulConfig.groovy]: `if (!agentEndpoint.contains(":"))`

File: clouddriver-consul/src/main/groovy/com/netflix/spinnaker/clouddriver/consul/config/ConsulConfig.groovy [hunk 2]
```diff
@@ -41,8 +41,12 @@ class ConsulConfig {
...
+      agentEndpoint = "${agentEndpoint}:${agentPort}"
```
Comment (duftler) [inline clouddriver-consul/src/main/groovy/com/netflix/spinnaker/clouddriver/consul/config/ConsulConfig.groovy]: `agentEndpoint += ":$agentPort"`

**Comment by duftler** [comment]
LGTM after comments.

**Pull request merged**